src/java.base/share/classes/sun/net/www/protocol/http/DigestAuthentication.java

```diff
@@ -427,6 +508,24 @@ public void checkResponse (String header, String method, URL url)
         checkResponse (header, method, url.getFile());
     }
 
+    private void validateAlgorithm(String algorithm) throws IOException {
+        if (getAuthType() == AuthCacheValue.Type.Server &&
+                getProtocolScheme().equals("https")) {
+            // HTTPS server authentication can use any algorithm
+            return;
+        }
+        algorithm = algorithm.toUpperCase();
+        if (disabledAlgs.contains(algorithm)) {
+            String msg = "Rejecting digest authentication with insecure algorithm: "
+                + algorithm;
+            if (logger.isLoggable(PlatformLogger.Level.INFO)) {
+                logger.info(msg + " This constraint may be relaxed by setting " +
+                     "the \"http.auth.digest.enabledDigestAlgs\" system property.");
+            }
```

Comment:
I suspect this is a typo and perhaps should have been `http.auth.digest.reEnabledAlgs`?

Reply:
Thanks. I'm going to change the name once more and will update it then.